test/libraries/WrappedSignatureLib.t.sol

```diff
@@ -60,4 +60,57 @@ contract WrappedSignatureLibTest is Test {
         assertEq(_arg3, arg3);
         assertEq(_arg4, arg4);
     }
+
+    function test_decodeSignatureWithHookData() public view {
+        bytes memory data = abi.encode(bytes(""), bytes(""));
+        (bytes memory _arg1, bytes memory _arg2) = decoder.decodeWithHookData(data);
+        assertEq(_arg1, bytes(""));
+        assertEq(_arg2, bytes(""));
+    }
+
+    function test_decodeSignatureWithHookData_incorrectlyEncodedSignature_reverts() public {
+        bytes memory data = abi.encode(bytes32(keccak256("test")), bytes(""));
```

Comment:
this is expected to error because the first word, which should be the length, is some bytes32 value when the signature is empty right?